### ISSUE TITLE
Issues about the LLVM OpenMP runtime system and its low-level threading library.

### ISSUE THREAD

**Issue by FuncJ** [llvm/llvm-project#60846]
Hi, Iwasaki et al. have written a paper, 'BOLT: Optimizing OpenMP Parallel Regions with User-Level Threads. ' They chose the LLVM OpenMP 7.0( https://openmp.llvm.org/) as a baseline OpenMP implementation and made improvements for optimizing the nested parallel region. Does LLVM OpenMP now adopt the strategies presented in the paper?

**Comment by shiltian** [comment]
No. We don't use user-level threads in libomp for now. It is always welcomed to have the patches submitted such that we can take a proper code review.
